src/store/modules/user/actions.ts

```diff
@@ -95,6 +95,14 @@ const actions: ActionTree<UserState, RootState> = {
       let stores = [] as any;
       if(storeResp.status === 200 && !hasError(storeResp) && storeResp.data.docs?.length > 0) {
         stores = [...storeResp.data.docs]
+        
+        resp.data.stores = [
+          ...(stores ? stores : []),
+          {
+            productStoreId: "",
+            storeName: "None"
```

Comment:
With the preorder app's use case, the user will get data specific to the product store or all the data, consider the products page if a product store is selected, app products belonging to the selected store are fetched if not all the products are fetched.

```suggestion
            storeName: "All"
```

Reply:
Yes, sir, understood.